client/src/main.ts

```diff
@@ -1,4 +1,38 @@
 import './styles/style.css';
+import { swapClassBetweenTwoElements } from './utils/helperfunctions';
 import { io } from 'socket.io-client';
 
 const socket = io('http://localhost:3000/');
+
+const usernameInput = document.getElementById('loginInput');
+const loginButton = document.getElementById('loginButton');
+const loginSection = document.getElementById('loginSection');
+const gameLobbySection = document.getElementById('gameLobbySection');
+
+/**
+ * Handles login for user
+ * Picket username is set in localstorage
+ * Displays error message if input is not filled
+ * Emits username to server using socket
+ * @param {Element | null} input
+ * @param {Element | null} loginSection
+ * @returns void
+ */
+function handleLoginOnClick(input: Element | null, loginSection: Element | null) {
```

Comment:
Funktionen handleLoginOnClick kan brytas ner till mindre funktioner för att följa principen om att funktioner bör göra endast en sak. Till exempel skulle man kunna ha en funktion för validering, en för att skicka användarnamn till servern och en för att för att byta mellan login-vyn och spellobbyn. Kanske overkill men tänker på att följa principen om att varje funktion bör göra endast en sak. :)

Reply:
Bra att ta i beaktning att en funktion följer en princip, men i slutändan är denna funktion för kort, 9 rader och i stort sett gör den bara en sak, loggar in. Dock hade vi haft säg avancerad validering med regex, större socket funktion (kan hända den blir större, sen), ska visa ett error meddalande så man kan dela upp dem i tre mindre separata funktioner. Så i fall funktionen byggs ut kan vi tänka på detta 👍